tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -1098,7 +1103,11 @@ def _call_load_weights(self, load_method, weights, weight_mapper):
             load_method(weights)
 
     def _init_max_seq_len(self):
-        inferred_max_seq_len = self.model.infer_max_seq_len()
+        if self.pytorch_backend_config.mm_encoder_only:
```

Comment:
Where is 32768 coming from? Can we address the TODO right away?

Reply:
Sure. I addressed it by add warnings and moved overideen `max_num_token` at beginning, see `mm_encoder.py`

Reply:
BTW, the default of `max_num_tokens`  in BuildConfig is 8192 - 32768 is 4x of it.